testsuite/forge.py

```diff
@@ -1251,6 +1251,7 @@ def test(
     github_run_id: Optional[str],
     github_step_summary: Optional[str],
 ) -> None:
+    forge_enable_failpoints = True
```

Comment:
this needs to be a string unfortunately

Reply:
see https://github.com/aptos-labs/aptos-core/pull/3890

Reply:
oh. it worked like this on some previous change. will update